modules/ROOT/pages/performance-and-impact.adoc

```diff
@@ -10,11 +10,61 @@ Based on typical usage, the expected CPU utilization is less than 5%. For more i
 
 Anypoint Monitoring asynchronously collects data, which does not block Mule runtime data processing.
 
+=== Gold and Platinum Subscriptions
+
+Gold and Platinum subscribers to Anypoint Platform receive modest data retention and feature limits:
+
+Logs::
+Log limits are documented in xref:runtime-manager::viewing-log-data.adoc#log-persistence[Runtime Manager].
+
+Application Metrics::
+30 days
+
+Enhanced API Metrics::
+30 days. The limit for data in the *Client Applications* tab is 1 hour.
+
+Basic Alerts::
+50 basic alerts
+
+////
+Gold and Platinum subscribers to Anypoint Platform receive
 * Minute Resolution for Metrics.
```

Comment:
Should minute resolution for metrics go somewhere? 🤔  Wasn't sure what that legacy text is trying to convey.

Reply:
Sure. We can lump this info together with the statement that we asynchronously gather metrics data, at a minute resolution.